environment_unix.yml

```diff
@@ -39,15 +42,17 @@ dependencies:
   - lmdb==0.9.22
   - lmdbxx==0.9.14
   # Python dependences
-  - python
+  - python =3.10 # Python 3.11 requires protobuf>=4
```

Comment:
What's the difference between `==<version>` and ` =<version>`?

Reply:
See https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html:

The fuzzy constraint numpy=1.11 matches 1.11, 1.11.0, 1.11.1, 1.11.2, 1.11.18, and so on.

The exact constraint numpy==1.11 matches 1.11, 1.11.0, 1.11.0.0, and so on.